keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1039,15 +1039,8 @@ If this RPC failed, or kubelet does not know if it failed or not, it
 MAY choose to call `NodePrepareResource` again, or choose to call
 `NodeUnprepareResource`.
 
-After a successful call, Kubelet MUST pass device names and kinds to
-the runtime through the CRI protocol.
-
-```
-<<[UNRESOLVED @bart0sh]>>
-CRI protocol may need to be extended for
-this purpose, e.g. device id can be added to the CRI Device structure.
-<<[/UNRESOLVED]>>
-```
+On a successful call this RPC should return set of CDI device ids,
```

Comment:
What is a "CDI device id"? That term is not used in https://github.com/container-orchestrated-devices/container-device-interface/blob/master/SPEC.md.

It's okay to be informal here as long as it is explained precisely below, but that's not currently the case.